packages/sdk/src/vaults/constants.ts

```diff
@@ -56,13 +56,26 @@ export const COW_SETTLEMENT_ADDRESS_LIST: Record<ChainId, string> = {
   [ChainId.ARBITRUM]: COW_SETTLEMENT_ADDRESS,
 };
 
-const STACKLY_SUBGRAPH_API_KEY =
+const API_BASE_URL = "https://gateway-arbitrum.network.thegraph.com/api";
+const SUBGRAPH_API_KEY =
   process.env.STACKLY_SUBGRAPH_API_KEY ?? "e7b7ff845e506590498946cd6bf83bf6";
 
+export const ETHEREUM_API_ENDPOINT_URL =
+  process.env.ETHEREUM_SUBRAPH_API_URL ??
```

Comment:
typo, same for the others

```suggestion
  process.env.ETHEREUM_SUBGRAPH_API_URL ??
```